Makefile

```diff
@@ -60,10 +60,10 @@ mypy:  ## Run mypy
 safety:  ## Run https://github.com/pyupio/safety
 	poetry run safety check --full-report
 
-publish: ## Release new version to PyPi
+publish: install  ## Release new version to PyPi
```

Comment:
I really didn't see the need for it. "install" is not really expensive. Normally it does nearly nothing. Otherwise you also wont be able to publish ;)